extension/content_script.js

```diff
@@ -18,7 +18,6 @@ var tracker = null;
 var timer = null;
 var speed = 10; // Base speed, not accounting for sentence length; adjustable w/ D/S
 var speed_bias = 500; // Minimum amount of speed spent on each sentence (in milliseconds)
-var speed_adj = 0; // Speed after it has been adjusted by sentence length
```

Comment:
don't need this. See calculateTrackerLife().
I'm trying to minimize the number of global variables and the bookkeeping cost of having to maintain and making sure they're consistent.

Reply:
Yep v good idea and thanks for identifying this as good candidate for eliminating global variable
LGTM